cypress/support/fragments/settings/remote-storage/configurations.js

```diff
@@ -65,12 +65,15 @@ function openCreateConfigurationForm() {
 
 const configurations = {
   DematicEMS: {
-    title: 'Dematic EMS',
+    title: REMOTE_STORAGE_PROVIDER_NAMES.Dematic_EMS,
```

Comment:
lets keep this consistent for other configuration titles too

Reply:
done